Deploy-CAM.ps1

```diff
@@ -2667,7 +2690,7 @@ function Write-Host-Warning() {
     param(
         $message
     )
-    Write-Host ("`n$message") -ForegroundColor Red -BackgroundColor Yellow
+    Write-Host ("`n$message") -ForegroundColor Red
```

Comment:
We all agree just to show Red Text?

Reply:
Backgroud Color is what it is. It is more readable